src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindOneAndUpdateCommand.java

```diff
@@ -17,7 +18,14 @@
 public record FindOneAndUpdateCommand(
     @Valid @JsonProperty("filter") FilterClause filterClause,
     @Valid @JsonProperty("update") UpdateClause updateClause,
-    @Nullable Options options)
+    @Valid @Nullable Options options)
     implements ReadCommand, Filterable {
-  public record Options() {}
+  public record Options(
+      @Valid
+          @Nullable
+          @Pattern(
+              regexp = "(after|before)",
+              message = "returnDocument value can only be before or after")
```

Comment:
surround before and after in (single?) quotes to indicate they are literal values

Reply:
Done.